### ISSUE TITLE
list command fails on albums whose name contains accented characters

### ISSUE THREAD

**Issue by copernico** [gingerlime/smugline#20]
```
C:\Users\******\Desktop\smugline-master>python smugline.py --api-key=*****  list > albums.txt
Password:
Traceback (most recent call last):
  File "smugline.py", line 317, in <module>
    smugline.list_albums()
  File "smugline.py", line 217, in list_albums
    print(album['Title'])
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe0' in position 7: ordinal not in range(128)
```

**Comment by gingerlime** [comment]
are you using python 2 or python 3? can you test it with python 3?

Not sure I'll have time to dig into this, but happy to get a PR if you want to take a stab at fixing it.

**Comment by copernico** [comment]
Python 2.7.12.
I might try on 3.x when I have some time.
Thanks for the nice script! I'm using it to reclaim my 60GB of stuff from Smugmug. It will take some time..... :-)

**Comment by gingerlime** [comment]
Thanks. Where are you moving it away to?

**Comment by copernico** [comment]
There you have my small PR :-)
NOTE: a similar treatment of printed strings might be needed elsewhere (but I did not try all use cases, I only need to download, and that's what I tested). This fix works fine in my scenario (I have hundreds of albums, and non ascii letters are quite common).

I'm moving to Amazon Photos, which I pay anyway (with Prime) and I want to simplify my home IT setup (and expenses) by removing some duplication.
